tests/load/locustfile.py

```diff
@@ -0,0 +1,405 @@
+from __future__ import absolute_import, print_function, unicode_literals
+
+import base64
+import json
+import time
+import uuid
+from contextlib import closing
+from urllib.parse import urljoin, urlparse
+
+from locust import HttpUser, TaskSet, events, task
+from websocket import create_connection
+from websocket._exceptions import WebSocketTimeoutException
+
+
+@events.init_command_line_parser.add_listener
+def _(parser):
+    parser.add_argument(
+        "--websocket_url",
+        type=str,
+        env_var="AUTOPUSH_WEBSOCKET_URL",
+        default="",
+        help="Server URL",
+    )
+    parser.add_argument(
+        "--endpoint_url",
+        type=str,
+        env_var="AUTOPUSH_ENDPOINT_URL",
+        default="",
+        help="Endpoint URL",
+    )
+
+
+class ConnectionTaskSet(TaskSet):
+    @task
+    def test_basic(self):
+        encrypted_data = "aLongStringOfEncryptedThings"
```

Comment:
For real load testing, I wonder if we shouldn't use blocks of ballast data?

(I appreciate having a test signifier prefix so that we can easily clean crap out if need be)

```
encrypted_data = "TestData" + ''.join([random.choice(string.ascii_letters + string.digits + "-_") for i in range(0,random.randint(1024,4096)-8)])
```

Reply:
Could you explain what this is doing? I am getting an error ```binascii.Error: Incorrect padding```.

Reply:
Ah! I missed that you're trying to decode that string. I'm not sure why. If anything, it would make more sense to have

```python
data = "TestData" + ''.join([random.choice(string.ascii_letters + string.digits ) for i in range(0,random.randint(1024,4096)-8)])
encrypted_data = base64.urlsafe_b64encode(data)
```

then later post (@62)
```python
       endpoint_res = self.client.post(
            url=endpoint_url,
            name="ENDPOINT test_basic",
            data=encrypted_data,
            headers=headers,
        )
```

The check would then be (@72)

```python
   assert data  == base64.urlsafe_b64decode(res["data"])
```

This matches (kinda) the actual behavior of the service. As a bonus it also potentially exercises a bug that we had regarding the actual max buffer size we were allowing (It was set too low due to a mistake around value encoding lengths)

Reply:
Thinking about this a bit more. 
I'm fine with keeping things as is for now, since the primary focus is getting this over into Locust. The final load tester could generate randomized payloads to exercise the target system (along with randomized TTLs). 

Just so things are noted and clear:
The server doesn't pay attention to the incoming payload. It does, however, convert it to base64 when sending it out due to limitations with the original websocket protocol and some mobile platforms. Instead of generating a fake base64 value, it's just as easy to provide user readable data and then do a

```python
base64.urlsafe_b64decode(res["data"] + ("==="[:len(res["data"]) % 4]))
```
(which appends padding to the end of the returned `res["data"]` string so that base64 can properly decode it.)